com.avaloq.tools.ddk.xtext.generator/src/com/avaloq/tools/ddk/xtext/generator/parser/antlr/IgnoreCaseString.java

```diff
@@ -0,0 +1,103 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Avaloq Evolution AG and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Avaloq Evolution AG - initial API and implementation
+ *******************************************************************************/
+
+package com.avaloq.tools.ddk.xtext.generator.parser.antlr;
+
+import com.google.common.base.Preconditions;
+
+
+/**
+ * Copied from {@code org.eclipse.xtext.generator.parser.antlr.ex.common.IgnoreCaseString}.
+ * Could not use the original because of the protected constructor.
+ */
+public class IgnoreCaseString implements CharSequence {
+
+  private final String value;
+
+  /**
+   * Creates a new instance of {@link IgnoreCaseString}.
+   *
+   * @param value
+   *          String value
+   */
+  public IgnoreCaseString(final String value) {
+    Preconditions.checkNotNull(value);
+    this.value = value;
+  }
+
+  /**
+   * {@inheritDoc}
+   */

```

Comment:
here and in the rest of the class, please remove the @inheritDoc which does not add any content

Reply:
ok.